libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1076,6 +1076,18 @@ AP_InertialSensor::detect_backends(void)
 
 #define ADD_BACKEND_INSTANCE(x, instance) if (instance == _backend_count) { ADD_BACKEND(x); }
 
+#define ADD_BACKEND_AUX(x, devid) do { \
```

Comment:
need a comment explaining what this does, how it relates to hwdef.dat, with an example